src/assessment.handlebars

```diff
@@ -73,7 +73,7 @@
       <h2 class="question-text" tabindex="-1">
 					<span id="category-{{sluggify question}}" class="activity" aria-hidden="true">{{category}}</span>
 					<span class="vh">Question:</span>
-					<em>{{question}}</em>
+					<em>{{{question}}}</em>
```

Comment:
nice! Might have security implications but no worries 'til we get hacked... just kiddin'

I think it's fine as this is not the compiled code. Would be a pretty big step to build and release a new version with dangerous vibes in - so I think ok.